apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/post-published-sharing-modal/index.tsx

```diff
@@ -0,0 +1,269 @@
+import { Modal, Button, ExternalLink } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect, useRef, useState, createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Icon, globe, link as linkIcon } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import classnames from 'classnames';
+import React from 'react';
+import ClipboardButton from 'calypso/components/forms/clipboard-button';
+import FormInputCheckbox from 'calypso/components/forms/form-checkbox';
+import FormLabel from 'calypso/components/forms/form-label';
+import SocialLogo from 'calypso/components/social-logo';
+import useShouldShowSellerCelebrationModal from '../../../dotcom-fse/lib/seller-celebration-modal/use-should-show-seller-celebration-modal';
+import useShouldShowVideoCelebrationModal from '../../../dotcom-fse/lib/video-celebration-modal/use-should-show-video-celebration-modal';
+import { selectors as wpcomWelcomeGuideSelectors } from '../store';
+import postPublishedImage from './images/illo-share.svg';
+import useSharingModalDismissed from './use-sharing-modal-dismissed';
+import type { SelectFromMap } from '@automattic/data-stores';
+import './style.scss';
+
+type WpcomWelcomeGuideSelectors = SelectFromMap< typeof wpcomWelcomeGuideSelectors >;
+type CoreEditorPlaceholder = {
+	getCurrentPost: ( ...args: unknown[] ) => { link: string; title: string };
+	getCurrentPostType: ( ...args: unknown[] ) => string;
+	isCurrentPostPublished: ( ...args: unknown[] ) => boolean;
+};
+const FB_APP_ID = '249643311490';
```

Comment:
Im not totally up to speed on the context here, but is this app ID something that should be set as an environment variable or hidden from the public eye in any way?

Reply:
@Addison-Stavlo nice catch! I think you're right.

All of the [other instances](fbhepr%2Sfrnepu%3Scebwrpg%3Qpnylcfb%26cebwrpg%3Qwrgcnpx%26cebwrpg%3Qjcpbz%26shyy%3Q249643311490%26qrsf%3Q%26ersf%3Q%26cngu%3Q%26uvfg%3Q%26glcr%3Q%26keq%3Q%26aa%3Q3%26a%3Q25%26fgneg%3Q25-og) I see of this ID are in PHP files.

On https://designschool.org/ (in the top right corner) I just link to `https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Fdesignschool.org%2F` without any JS includes and it works. Wonder if we could do something similar here (but still show it in a popup vs. full screen).

Reply:
Sorry. My A8c github extension is rewriting the URL to prevent it from leaking. I'll post the link in our Slack channel.

> I wasn't sure if this was something that should be hidden from a public repo or not.

I'll let Eru weigh in when he's online.

Reply:
I think I may have been confusing this with the App Secret.

> I'll let Eru weigh in when he's online.

👍